packages/plugins/rubygem-hammer_cli_foreman_puppet/rubygem-hammer_cli_foreman_puppet.spec

```
@@ -0,0 +1,95 @@
+# template: hammer_plugin
+%{?scl:%scl_package rubygem-%{gem_name}}
+%{!?scl:%global pkg_name %{name}}
+
+%global gem_name hammer_cli_foreman_puppet
+%global plugin_name foreman_puppet
+
+%{!?_root_sysconfdir:%global _root_sysconfdir %{_sysconfdir}}
+%global hammer_confdir %{_root_sysconfdir}/hammer
+
+Name: %{?scl_prefix}rubygem-%{gem_name}
+Version: 0.0.2
+Release: 1%{?foremandist}%{?dist}
+Summary: Foreman Puppet plugin for Hammer CLI
+Group: Development/Languages
+License: GPLv3
+URL: https://github.com/theforeman/hammer-cli-foreman-puppet
+Source0: https://rubygems.org/gems/%{gem_name}-%{version}.gem
+
+# start specfile generated dependencies
+Requires: %{?scl_prefix_ruby}ruby(release)
+Requires: %{?scl_prefix_ruby}ruby
+Requires: %{?scl_prefix_ruby}ruby(rubygems)
+Requires: %{?scl_prefix}rubygem(hammer_cli_foreman) >= 3.0.0
+Requires: %{?scl_prefix}rubygem(hammer_cli_foreman) < 4.0.0
+BuildRequires: %{?scl_prefix_ruby}ruby(release)
+BuildRequires: %{?scl_prefix_ruby}ruby
+BuildRequires: %{?scl_prefix_ruby}rubygems-devel
+BuildArch: noarch
+Provides: %{?scl_prefix}rubygem(%{gem_name}) = %{version}
+# end specfile generated dependencies
+
+%description
+Foreman Puppet plugin for Hammer CLI.
+
+
+%package doc
+Summary: Documentation for %{pkg_name}
+Group: Documentation
+Requires: %{?scl_prefix}%{pkg_name} = %{version}-%{release}
+BuildArch: noarch
+
+%description doc
+Documentation for %{pkg_name}.
+
+%prep
+%{?scl:scl enable %{scl} - << \EOF}
+gem unpack %{SOURCE0}
+%{?scl:EOF}
+
+%setup -q -D -T -n  %{gem_name}-%{version}
+
+%{?scl:scl enable %{scl} - << \EOF}
+gem spec %{SOURCE0} -l --ruby > %{gem_name}.gemspec
```

You can probably patch the gemspec like this:
```suggestion
gem spec %{SOURCE0} -l --ruby > %{gem_name}.gemspec
sed -i 's/.pre.develop//' %{gem_name}.gemspec
```
```